package.json

```diff
@@ -8,24 +8,22 @@
     "url": "https://github.com/latticexyz/mud.git"
   },
   "scripts": {
+    "// TODO make this use turob instead?  an we get rid of lerna?  link:packages": "lerna exec 'pnpm link --global'",
+    "// TODO what is this one?   Can we get rid of lerna?  entry:dist": "lerna run prepack",
+    "// TODO what is this one?   Can we get rid of lerna?  entry:src": "lerna run postpack",
+    "// TODO what is this one?release:yalc": "(pnpm entry:dist && lerna exec npx yalc push); pnpm entry:src",
     "build": "turbo run build",
     "clean": "turbo run clean",
     "dev": "turbo run dev --concurrency 100",
     "docs": "pnpm recursive run docs && pnpm prettier --write '**/*.md' && pnpm run --filter docs build:prepare && pnpm run --filter docs build",
-    "entry:dist": "lerna run prepack",
-    "entry:src": "lerna run postpack",
     "foundryup": "curl -L https://foundry.paradigm.xyz | bash && bash ~/.foundry/bin/foundryup",
     "gas-report": "pnpm recursive run gas-report",
-    "link:packages": "lerna exec 'pnpm link --global'",
     "lint": "pnpm prettier:check && eslint . --ext .ts --ext .tsx",
     "prepare": "husky install && (forge --version || pnpm foundryup)",
     "prettier": "prettier --write '**/*.{ts,tsx,css,md,mdx,sol}'",
     "prettier:check": "prettier --check '**/*.{ts,tsx,css,md,sol}'",
-    "release": "pnpm install && lerna publish --no-private --force-publish",
-    "release:canary": "lerna publish premajor --preid alpha.1 --canary --no-private --force-publish",
```

Comment:
i think we can get rid of the canary releases if we have prereleases (i'd prefer a regular prerelease that includes a changelog over a canary release with every push to main, and this also allows us to get rid of lerna)

Reply:
I don't have a strong preference here but let me give you my reasoning why I included a canary release.  Basically it's the idea of continous delivery. My default is try to publish (or dry-run publish) every single pr so there is confidence the deployment process is working on all changes. I could not figure out a way to do a dry run with changesets which is why I included this.

Though this is a general principle I follow by default, changesets are pretty bullet proof so I don't think it actually matters in this case. Especially since I expect ya'll will not be waterfalling releases but publishing prereleases often.

Reply:
One more piece of context, I shouldn't call this canary because that's not what they really are.  So a prerelease if you install `pnpm install @lattice-xyz/cli@*` or @next it will install the latest prerelease even if one of these previews got published.  `pnpm install @lattice-xyz/cli` or `@latest` will install only the latest stable release.  This release is kinda a one off without an actual tag I believe

Reply:
Looks like changesets calls them snapshot releases not canary releases.  https://github.com/changesets/changesets/blob/main/docs/snapshot-releases.md.

Should name them snapshots for consistency if we add them

Reply:
ohh okay gotscha! yeah so if these snapshots are just a way to explicitly install a commit version, but `@next` going to the latest prerelease and `@latest` going to the stable release, then I agree with this approach!

Reply:
Oops I just nuked the canary action

Reply:
brought it back